modules/end-user-guide/pages/configuring-an-existing-workspace-to-use-intellij-idea.adoc

```diff
@@ -0,0 +1,5 @@
+[id="configuring-an-existing-workspace-to-use-intellij-idea"]
+:navtitle: Configuring an existing workspace to use IntelliJ IDEA
+:keywords: end-user-guide, jetbrains-intellij-idea-community-edition
+
```

Comment:
Not sure.. just check if this is an assembly, then you may want to include toc::[] underneath the context definition to add a TOC.

Reply:
As we use them in che-docs, the files in the `pages` directory contain only metadata specific to Antora and an include statement to a modular file.